test/conformance/kuttl/generate/clusterpolicy/cornercases/pod-restart-on-cm-update/06-check-restart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl get po -n kube-state-metrics
+  - script: "if [ \"$(kubectl get po -n kube-state-metrics | awk 'NR==2{print $1}')\" != \"$(cat pod-name.txt)\" ];then exit;else (exit 1);fi"
```

Comment:
This is good, but there's probably a simpler way to go about this. Fine for now.